packages/astro/src/core/ssr/css.ts

```diff
@@ -1,15 +1,18 @@
 import type vite from '../../../vendor/vite';
 
+import { fileURLToPath } from 'url';
 import path from 'path';
+import slash from 'slash';
 
 // https://vitejs.dev/guide/features.html#css-pre-processors
 export const STYLE_EXTENSIONS = new Set(['.css', '.pcss', '.scss', '.sass', '.styl', '.stylus', '.less']);
 
 /** find unloaded styles */
-export function getStylesForID(id: string, viteServer: vite.ViteDevServer): Set<string> {
+export function getStylesForID(filePath: URL, viteServer: vite.ViteDevServer): Set<string> {
```

Comment:
Now that its signature has changed, should this be renamed to something like `getStylesForURL`?